app/views/api/v1/restaurants/index.json.jbuilder

```diff
@@ -0,0 +1,6 @@
+json.entries do
+    json.array! @restaurants.each do |restaurant|
```

Comment:
To create the correct endpoint output we needed this here, thomas helped us with this. Probably not an answer but something :)